api/EXT_sRGB.json

```diff
@@ -5,40 +5,40 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/EXT_sRGB",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "40"
           },
           "chrome_android": {
-            "version_added": true
+            "version_added": "40"
           },
           "edge": {
-            "version_added": "79"
+            "version_added": false
           },
           "firefox": {
-            "version_added": "28"
+            "version_added": "58"
```

Comment:
This is EXT_sRGB, and https://bugzilla.mozilla.org/show_bug.cgi?id=843668#c56 suggests that Firefox 28 was correct, but https://bugzilla.mozilla.org/show_bug.cgi?id=1403798#c5 suggests that it was fixed on Windows in Firefox 58.

WebGL is more OS- and hardware-dependent than most web APIs. This and the WebGL 1 vs 2 issues does make me doubt the validity of the results overall, not just where a real version number has been changed.

What do you think about collecting results from macOS as well to get some more confidence in the results?

Reply:
It looks like it _is_ an OS-specific thing.  It isn't available in Windows until Firefox 58, but on macOS, 28 returns `true`.  I can certainly update the collector's Selenium script to obtain macOS results as well!

Reply:
That would be great, so that we don't set some versions numbers here to earlier than the real cross-platform support.